client/blocks/jitm/templates/modal-style.scss

```
@@ -37,6 +37,20 @@ $wpcom-modal-footer-height: 30px + ( $wpcom-modal-footer-padding-v * 2 );
 	.components-modal__header {
 		display: flex;
 		position: absolute;
+
+		button.components-button {
+			span {
+				.components-popover.components-tooltip {
+					left: unset!important;
+					right: -48px;
+
+					.components-popover__content {
+						overflow: visible!important;
+						max-width: unset!important;
```

Could we add white spaces so it's easier to read (I'm actually surprised that it even works)?

```suggestion
					left: unset !important;
					right: -48px;

					.components-popover__content {
						overflow: visible !important;
						max-width: unset !important;
```

---

I always used !important without spaces 😄 I'm changing it